docker.go

```diff
@@ -90,6 +98,37 @@ func (container *Container) EmptyVolumes() *Container {
 	return container
 }
 
+// Define external elements for a volume
+//      name: the name of the volume
+//      provider: the provider of the volume (e.g. dvdi)
+func (v *Volume) ExternalVolume(name, provider string) *ExternalVolume {
+	ev := ExternalVolume{
+		Name:     name,
+		Provider: provider,
+	}
+	v.External = &ev
+	return &ev
+}
+
```

Comment:
Would be great if we could also have an `EmptyVolume()` method. `TestExternalVolume()` should then be extended by a call to that function and a check that the external volume is actually reset.